srcpkgs/OpenJK/files/README.voidlinux

```diff
@@ -0,0 +1,11 @@
+OpenJK requires a copy of Jedi Knight: Jedi Academy.
+You can also use this for Jedi Outcast (singleplayer only), which requires a copy as well.
+The default folder that the game will search for files is
+
+	~/.local/share/openjk
```

Comment:
I would be more verbose about what game does use what directory in `~/.local/share` like Jedi Academy -> `openjk` and Jedi Outcast -> `openjo` (If I remember correctly)